### PR TITLE
Added scenario for CF Initial setup verification.

### PR DESCRIPTION
**Pull request by archyufa**
This scenario only checks if CF is setup correctly prior to migration
and all prerequisites for source and destination clouds have been met.

### CODE REVIEW

**Comment by roman-verchikov** [comment]
Looks good to me, +1

File: scenario/cf_prechecks.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Initial checks only, to ensure CF is setup correctly prior to migration.
+# This scenario checks includes: SSH connectivity to compute nodes,
+# DB access, RabbitMQ connectivity, required bandwidth verification,
+# Opentack API functionality on source  and destination clouds.
+# As well as network verification such as Segmentation IDs and
+# Floating IP overlap checks. And no VMs spawned directly in external network.
```
Comment (miarmak) [inline scenario/cf_prechecks.yaml]: And also we check subnets overlapping there :wink:

File: scenario/cf_prechecks.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+namespace:
```
Comment (miarmak) [inline scenario/cf_prechecks.yaml]: I guess we can remove `namespace` block.

File: scenario/cf_prechecks.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          - act_check_image_api_dst: True
+          - act_check_compute_api_dst: True
+          - act_check_storage_api_dst: True
+          - act_check_neutron_api_dst: True
+          - check_dst_ssh_access: True
+          - act_check_bandwidth_dst: True
+          - check_dst_sql: True
+          - check_dst_rabbit: True
+      - check_networks: True
+rollback:
```
Comment (miarmak) [inline scenario/cf_prechecks.yaml]: And this

File: scenario/cf_prechecks.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+process:
```
Comment (miarmak) [inline scenario/cf_prechecks.yaml]: And this

**Comment by miarmak** [comment]
And also, I guess there is no need to specify `cf` in the scenario name. I would suggest simply using `scenario/precheck.yaml`.

**Comment by miarmak** [comment]
One more thing: please add Jira ticket number in the beginning of commit message (and don't forget about it's length. :wink: The first line limited to 50 characters)

**Comment by miarmak** [comment]
Looks good, thanks!
+1

**Comment by roman-verchikov** [comment]
can be merged.

**Pull request merged**